tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks/WriteBuildOutputProps.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Microsoft.Build.Framework;
```

Comment:
Do we require copyright headers in this repo?

Reply:
yes

Reply:
I fixed this for all the task C# files, and removed the ones that don't appear to be used anymore.